src/kili/adapters/kili_api_gateway/organization/operations.py

```diff
@@ -0,0 +1,74 @@
+"""Collection of Organization's related GraphQL queries and mutations."""
+
+ORGANIZATION_FRAGMENT = """
+id
+"""
+
+
+def get_create_organization_mutation(fragment: str) -> str:
+    """Return the GraphQL createOrganization mutation."""
+    return f"""
+mutation(
+    $data: CreateOrganizationData!
+) {{
+  data: createOrganization(
+    data: $data
+  ) {{
+    {fragment}
+  }}
+}}
+"""
+
+
+def get_list_organizations_query(fragment: str) -> str:
+    """Return the GraphQL organizations query."""
+    return f"""
+        query organizations($where: OrganizationWhere!, $first: PageSize!, $skip: Int!) {{
+            data: organizations(where: $where, first: $first, skip: $skip) {{
+                {fragment}
+            }}
+        }}
+        """
+
+
+def get_count_organizations_query() -> str:
+    """Return the GraphQL countOrganizations query."""
+    return """
+        query countOrganizations($where: OrganizationWhere!) {
+        data: countOrganizations(where: $where)
+        }
+    """
+
+
+def get_organization_metrics_query() -> str:
+    """Return the GraphQL getOrganizationMetrics query."""
+    return """
+    query organizationMetrics($where: OrganizationMetricsWhere!) {
+        data: organizationMetrics(where: $where) {
+            numberOfAnnotations
+            numberOfHours
+            numberOfLabeledAssets
+        }
+    }
+    """
+
+
+def get_update_properties_in_organization(fragment: str) -> str:
+    """Return the GraphQL updatePropertiesInOrganization mutation."""
+    return f"""
+mutation(
+    $id: ID!
+    $name: String
+    $license: String
```

Comment:
we could pass the data and where directly here, instead of unpacking the data and where